src/coverage/format.js

```diff
@@ -0,0 +1,145 @@
+const pad = require('pad')
+const { basename } = require('path')
+
+const ALL_FILES_PATH = '*'
+
+const ERROR_THRESHOLD = 50
+const WARN_THRESHOLD = 80
+
+const getEmoji = (percent) => {
+  if (percent === 0) {
+    return '❌'
```

Comment:
I suppose these could be customizable too in a .settings file in home, if that feature is added.